scripts/run_e2e_tests.py

```diff
@@ -467,6 +470,8 @@ def main(args=None):
     if not parsed_args.skip_build:
         build_js_files(
             dev_mode, deparallelize_terser=parsed_args.deparallelize_terser)
+    else:
+        build.modify_constants(prod_env=not dev_mode, maintenance_mode=False)
```

Comment:
Why do we need this change here?

Reply:
Since now we always reset the constants to default values after the suite is run, we need to set them again when different suite is run.